_includes/types/uri.md

```diff
@@ -16,6 +16,13 @@ We also maintain a node module to parse a string into `scheme`, `authority`, `pa
 
 Many of the interfaces contain fields that correspond to the URI of a document. For clarity, the type of such a field is declared as a `DocumentUri`. Over the wire, it will still be transferred as a string, but this guarantees that the contents of that string can be parsed as a valid URI.
 
+Care should be taken to handle encoding in URIs. For example, some clients (such as VS Code) may encode colons in drive letters while others do not. The URIs below are both valid, but clients and servers should be consistent with the form they use themselves to ensure the other party doesn't interpret them as distinct URIs. Clients and servers should not assume that each other are encoding the same way (for example a client encoding colons in drive letters cannot assume server responses will have encoded colons).
+
+```
+file:///c:/project/readme.md
```

Comment:
Maybe it would be better to use an uppercase drive letter in this example? Otherwise it could give the impression that drive letters must be lowercase, if both of the examples use a lowercase letter. Windows itself uses uppercase drive letters (almost) everywhere in the UI, so it seems more natural to use uppercase, and I remember bugs in the past with servers and with clients that expected a certain case for the drive letter.

Reply:
I agree with that suggestion @DanTup. May be best is to uppercase the drive letter only in one example.

Reply:
Done!